.travis.yml

```diff
@@ -23,11 +23,7 @@ install:
 
 addons:
   apt:
-    sources:
-      - sourceline: 'deb https://dl.bintray.com/sbt/debian /'
-        key_url: 'https://bintray.com/user/downloadSubjectPublicKey?username=sbt'
     packages:
-      - sbt=0.13.16
```

Comment:
APT is grabbing sbt here https://travis-ci.org/sbt/website/builds/263994877#L1317

```
Setting up sbt (0.13.16) ...
Creating system group: sbt
Creating system user: sbt in sbt with sbt daemon-user and shell /bin/false
Getting org.scala-sbt sbt 0.13.16  (this may take some time)...
downloading https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt/0.13.16/jars/sbt.jar ...
	[SUCCESSFUL ] org.scala-sbt#sbt;0.13.16!sbt.jar (1344ms)
```

Maybe sbt-extra is in earlier position of the path?